sticht/slack.py

```diff
@@ -267,22 +275,56 @@ def slack_api_call(self, *args, **kwargs):
                 log.debug(f'Exception encountered when making Slack api call: {e}')
                 return {'ok': False, 'error': f'{type(e).__name__}: {e}'}
 
+    def slack_post_message(self, **kwargs):
```

Comment:
Are you sure this function needs to post to a message *and* invite users all the time?

What if instead the calling function was responsible for inviting them once?

Because the calling function will know "who" to invite, and you won't have to extract the users from it in the first place 
(tell, don't ask principle)

Reply:
Created a new function invite at the user's discretion, instead of every time we post a message.